snapcraft/cli/_errors.py

```diff
@@ -20,6 +20,19 @@
 from . import echo
 from snapcraft.internal import errors
 
+import click
+# raven is not available on 16.04
+try:
+    from raven import Client as RavenClient
+    from raven.transport import RequestsHTTPTransport
+except ImportError:
+    RavenClient = None
+
+_SEND_TO_SENTRY = (
+    'Sorry, Snapcraft had an internal error.\n'
+    'To help us improve, would you like to send error data?'
```

Comment:
This message raises a couple of questions in my head:

- What data is this going to send?
- Will this create a bug report a la apport or ubuntu-bug or something else?

That is to say, perhaps the message can be more helpful in telling the user what is going to happen with regard to privacy concerns and what Snapcraft is going to do.